python/lsst/jointcal/jointcal.py

```diff
@@ -40,23 +40,45 @@ class JointcalRunner(pipeBase.TaskRunner):
 
     @staticmethod
     def getTargetList(parsedCmd, **kwargs):
+        """
+        Return a list tuples per tract: (dataRefs, kwargs).
+
+        Jointcal operates on lists of dataRefs simultaneously.
+        """
+        kwargs['profile_jointcal'] = parsedCmd.profile_jointcal
+
         # organize data IDs by tract
         refListDict = {}
         for ref in parsedCmd.id.refList:
             refListDict.setdefault(ref.dataId["tract"], []).append(ref)
         # we call run() once with each tract
-        return [(refListDict[tract],
-                 tract
-                 ) for tract in sorted(refListDict.keys())]
+        result = [(refListDict[tract], kwargs) for tract in sorted(refListDict.keys())]
+        return result
 
     def __call__(self, args):
+        """
+        @param args     Arguments for Task.run()
+
+        @return
+        - None if self.doReturnResults is False
+        - A pipe.base.Struct containing these fields if self.doReturnResults is True:
+            - dataRef: the provided data references, with update post-fit WCS's.
+        """
         task = self.TaskClass(config=self.config, log=self.log)
-        task.run(*args)
+        dataRefList, kwargs = args
+        result = task.run(dataRefList, **kwargs)
+        if self.doReturnResults:
+            return pipeBase.Struct(result=result)
 
 
 class JointcalConfig(pexConfig.Config):
     """Config for jointcalTask"""
 
+    coaddName = pexConfig.Field(
+        doc = "Type of coadd",
```

Comment:
Some kind of example in the doc string is helpful, e.g. "Coadd name: typically one of deep or goodSeeing."